gorgonia.go

```diff
@@ -266,6 +266,10 @@ func UnsafeLet(n *Node, be interface{}) error {
 		}
 
 	case Value:
+		if !n.Shape().Eq(v.Shape()) {
+			return fmt.Errorf("Unable to let %v. Expected Shapes to be equal. Got %v and %v instead", n.name, n.Shape(), v.Shape())
```

Comment:
Wondering if you could change the error msg to be clearer? Specifically the last sentence: "Node's expected shape is %v. Got %v instead"

Reply:
done, tests are running